### PR TITLE
chore: add multibuild

### PR DESCRIPTION
**Pull request by henryiii**
This adds multibuild, pre-req for #910. CC @matthew-brett - better description welcome (we don't have much of a description on cibuildwheel either, @pypa/cibuildwheel-team).

### CODE REVIEW

File: source/key_projects.rst [hunk 1]
```diff
@@ -55,9 +55,8 @@ cibuildwheel
 `Discussions <https://github.com/pypa/cibuildwheel/discussions>`__ |
 `Discord #cibuildwheel <https://discord.com/invite/pypa>`__
 
-``cibuildwheel`` is a package that builds wheels for all common platforms and
-Python versions on most CI systems.
-
+``cibuildwheel`` is a Python package that builds wheels for all common
```
Comment (bhrutledge) [inline source/key_projects.rst:58]:
```suggestion
``cibuildwheel`` is a Python package that builds :term:`wheels <Wheels>` for all common
```

File: source/key_projects.rst [hunk 2]
```diff
@@ -458,6 +457,16 @@ dependencies for, and publish packages to PyPI. Under the hood, it
...
 install packages.
 
+.. _multibuild:
+
+multibuild
+==========
+
+`GitHub <https://github.com/matthew-brett/multibuild>`__
+
+Multibuild is a set of CI scripts for building and testing Python Wheels for
```
Comment (bhrutledge) [inline source/key_projects.rst:467]:
```suggestion
Multibuild is a set of CI scripts for building and testing Python :term:`wheels <Wheel>` for
```

**Review by bhrutledge — COMMENTED** [review]
This might be pedantic, but "wheel" seems worth defining.

**Comment by bhrutledge** [comment]
Also, FWIW, I think it would have been okay to add this to #910.

**Comment by henryiii** [comment]
> Also, FWIW, I think it would have been okay to add this to #910.

I've [been told before](https://github.com/pypa/packaging.python.org/pull/823#issuecomment-774536521) by @webknjaz  it's better to keep things separate, so I've tried to do that. If one part blocks, the other can still go in.

File: source/key_projects.rst [hunk 3]
```diff
@@ -55,9 +55,8 @@ cibuildwheel
 `Discussions <https://github.com/pypa/cibuildwheel/discussions>`__ |
 `Discord #cibuildwheel <https://discord.com/invite/pypa>`__
 
-``cibuildwheel`` is a package that builds wheels for all common platforms and
-Python versions on most CI systems.
-
+``cibuildwheel`` is a Python package that builds :term:`wheels <Wheels>` for all common
```
Comment (henryiii) [inline source/key_projects.rst:58]:
```suggestion
``cibuildwheel`` is a Python package that builds :term:`wheels <Wheel>` for all common
```

**Review by bhrutledge — APPROVED** [review]
> it's better to keep things separate,

Agreed. In this case, though, it seems like these changes are tightly-coupled to the changes in #910, and IMHO would be more easily reviewed together. I don't think that would fall under @webknjaz's qualifications for breaking it up:

> I find it easier to review/approve separate unrelated changes so if one of the updates wouldn't block others from being merged. Usually when the title contains references to many things, has "and" in it, or is too vague within the 50 char limit, it's a sign that the PR is too broad and when you can describe its separate parts with a few specific titles, it's a good idea to have a PR per such a title. 

But I'm just being nit-picky while learning the ropes as a gardener for this project. 😄

**Comment by pradyunsg** [comment]
> I've [been told before](https://github.com/pypa/packaging.python.org/pull/823#issuecomment-774536521) by @webknjaz it's better to keep things separate, so I've tried to do that.

IMO, this is a balancing act. It's possible to go too-far with breaking up PRs -- having a bias toward smaller is good (easier to review + iterate), as long as there's no dependency between the various PRs. If there's dependencies in the changes, it's much better to just combine the changes into a single PR that can be reviewed as a whole.

**Review by pradyunsg — APPROVED** [review]
Thanks @henryiii for the PR and @bhrutledge for the thoughtful reviews! ^>^

**Pull request merged**